lib/galaxy/tool_util/toolbox/base.py

```diff
@@ -24,6 +24,7 @@
     NullDependencyManager,
 )
 from galaxy.tool_util.loader_directory import looks_like_a_tool
+from galaxy.tools.special_tools import load_lib_tools
```

Comment:
You cannot import stuff from `galaxy.tools` in `galaxy.tool_util`, otherwise the package hierarchy breaks. Can you please move `lib/galaxy/tools/special_tools.py` to `lib/galaxy/tool_util/special_tools.py` and update the import?

Maybe the `load_lib_tools` changes should go to dev and keep here only the `load_builtin_converters` part?